### PR TITLE
Add hash_on cookie load balancing docs

### PR DESCRIPTION
**Pull request by PepperTeasdale**
### Summary
Add documentation for new feature in [kong admin API](https://github.com/Kong/kong/pull/3472)

### CODE REVIEW

File: app/docs/0.14.x/admin-api.md [hunk 1]
```diff
@@ -84,10 +84,12 @@ upstream_body: |
...
+    `hash_on`<br>*optional* | What to use as hashing input: `none`, `consumer`, `ip`, `header`, or `cookie` (defaults to `none` resulting in a weighted-round-robin scheme).
+    `hash_fallback`<br>*optional* | What to use as hashing input if the primary `hash_on` does not return a hash (eg. header is missing, or no consumer identified): `none`, `consumer`, `ip`, `header`, or `cookie` (defaults to `none`, not available if `hash_on` is set to `cookie`).
     `hash_on_header`<br>*semi-optional* | The header name to take the value from as hash input (only required when `hash_on` is set to `header`).
     `hash_fallback_header`<br>*semi-optional* | The header name to take the value from as hash input (only required when `hash_fallback` is set to `header`).
+    `hash_on_cookie`<br>*semi-optional* | The cookie name to take the value from as hash input (only required when `hash_on` or `hash_fallback` is set to `cookie`). If the specified cookie is not in the request, Kong will generate a value and set in the response.
```
Comment (Tieske) [inline app/docs/0.14.x/admin-api.md]: > and set in the response.

not proper?

**Comment by Tieske** [comment]
seems it also needs some conflicts to be resolved

**Comment by PepperTeasdale** [comment]
I had tried to resolve the conflicts in the github UI, but the merge looked funny. Cleaned the commit history up and addressed that last comment.

**Comment by Tieske** [comment]
@hishamhm  @thibaultcha either of you want to review and merge?

**Comment by thibaultcha** [comment]
I will take a look, always; there is no rush since this is scheduled for 0.14 which we will be picking a date for.

**Comment by thibaultcha** [comment]
Manually merged, thank you @PepperTeasdale!

**Pull request closed without merging**